src/components/Filter/Filter.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import Filter from './Filter';
+
+export default {
+  title: 'Components/Filter',
+  component: Filter
+};
+
+export const TestFilter = () => (
+  <BrowserRouter>
+    <Filter />
+  </BrowserRouter>
```

Comment:
Would you clarify the purpose of wrapping the filter in the router here? It's academic - it's only Storybook, after all. But I am curious.

Reply:
Well when we do something again  and again then we just put some unnecessary code into next work:D
Its not important to have router here as we use router inside proper web page. but it doesnt damage the result.

Reply:
I copied the code from somewhere else but didn't quite understand why it was there. Should've queried it!